packages/flutter/lib/src/widgets/image.dart

```diff
@@ -472,9 +472,10 @@ class Image extends StatefulWidget {
   /// If non-null, this color is blended with each image pixel using [colorBlendMode].
   final Color color;
 
-  /// Used to set the filterQuality of the image
-  /// Use the "low" quality setting to scale the image, which corresponds to
-  /// bilinear interpolation, rather than the default "none" which corresponds
+  /// Used to set the [FilterQuality] of the image.
+  /// 
```

Comment:
I believe Cirrus is mad at the whitespace at the end of this and the next line.

(It also looks like the whitespace check is broken as it seems to have thrown an error instead of printing the offending lines)